toolium/utils/dataset.py

```diff
@@ -84,10 +84,13 @@ def replace_param(param, language='es', infer_param_type=True):
         [STR:xxxx] Cast xxxx to a string
         [INT:xxxx] Cast xxxx to an int
         [FLOAT:xxxx] Cast xxxx to a float
+        [ROUND:xxxx::2] String float with the expected decimals
```

Comment:
Como no es una conversión de tipos, mejor al final de la sección de strings generados.
```suggestion
        [ROUND:xxxx::y] Generates a string from a float number (xxxx) with the indicated number of decimals (y)
        [STR:xxxx] Cast xxxx to a string
        [INT:xxxx] Cast xxxx to an int
        [FLOAT:xxxx] Cast xxxx to a float
```